.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+Thank you for your Pull Request! We have developed this task checklist to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your blog post.
+
+Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.
+
+- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
+- [ ] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style
+- [ ] Chose (possibly several) `tag(s)` or categories from this list: `c("metadata", "submission", "qc", "ADaMs", "SDTMs", "community", "conferences", "admiral", "roak", "xportr", "metatools", "metacore")` for your blog post
+- [ ] Blog post is under a 10 minute read 
+- [ ] Blog post is *short*, *personalized*, *reproducible* and *readable*
+- [ ] Link the issue in the Development Section on the right hand side.
```

Comment:
```suggestion
```
Isn't this already covered by the first checkbox?

Reply:
Yeah you're right. maybe this sentence could be appended on the first checklist item then? Something along the lines of "and make sure the corresponding issue is linked in the Development section on the right hand side".